docs/csharp/roslyn-sdk/get-started/semantic-analysis.md

```diff
@@ -112,7 +112,7 @@ To finish this tutorial, let's build a LINQ query that creates a sequence of all
 
 [!code-csharp[Access the sequence of members on the string type](../../../../samples/csharp/roslyn-sdk/SemanticQuickStart/Program.cs#9 "Access the sequence of members on the string type.")]
 
-That source sequence conatins all members, including properties and fields, so filter it using the <xref:System.Collections.Immutable.ImmutableArray%601.OfType%2A?displayProperty=nameWithType> method to find elements that are <xref:Microsoft.CodeAnalysis.IMethodSymbol?diplayProperty=nameWithType> objects:
+That source sequence conatins all members, including properties and fields, so filter it using the <xref:System.Collections.Immutable.ImmutableArray%601.OfType%2A?displayProperty=nameWithType> method to find elements that are <xref:Microsoft.CodeAnalysis.IMethodSymbol?displayProperty=nameWithType> objects:
```

Comment:
Should the build system be catching the incorrect query params in the xref?

Reply:
I would have thought so.  Ping @meganbradley to check.  Should this have generated a warning (before the fix in this PR)?